Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -208,68 +202,7 @@ class CustomerIOImplementation: CustomerIOInstance {
      is no active customer, this will fail to register the device
      */
     public func registerDeviceToken(_ deviceToken: String) {
-        addDeviceAttributes(deviceToken: deviceToken)
-    }
-
-    /**
-     Adds device default and custom attributes and registers device token.
-     */
-    // TODO: Segment doesn't provide this method by default needs to get added
-    private func addDeviceAttributes(deviceToken: String, customAttributes: [String: Any] = [:]) {
```

Comment:
With `DataPipelineImplementation` handling all this stuff, we may completely remove `CustomerIOImplementation`, but doing it in phases so it is easier for us to make sure we aren't ignoring anything unknowingly.

Reply:
Update: We will be keeping `CustomerIOImplementation` as it will help us keep the same experience for existing users without impacting customer coming from Segment.